src/main/java/com/checkmarx/flow/service/BugTrackerEventTrigger.java

```diff
@@ -61,6 +61,9 @@
                 eventsWereTriggered = false;
                 break; // No action is needed
 
+            case NONE:
```

Comment:
You can put `case NONE:` directly below `case CUSTOM:`

Reply:
moved under Custom